README.md

```diff
@@ -20,7 +20,7 @@
 | [num-traits](https://docs.rs/num-traits/0.2.15)                                    | 0.2.15  |
 | [pyth-sdk](https://docs.rs/pyth-sdk/0.7.0)                                         | 0.7.0   |
 | [pyth-sdk-solana](https://docs.rs/pyth-sdk-solana/0.7.0)                           | 0.7.0   |
-| [solana-program](https://docs.rs/solana-program/1.14.12)                           | 1.14.12 |
+| [solana-program](https://docs.rs/solana-program/1.14.16)                           | 1.14.16 |
```

Comment:
One last thing, Playground still uses `solana-program` 1.14.12 and this PR does not update it(program crates don't live in this repository).

Can merge it after you restore this file.